### PR TITLE
6550 iOS Http/https flag

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-6550

Private changes: https://github.com/Swrve/swrve-sdk/pull/735

### CODE REVIEW

File: Sdk/Track/Swrve.h [hunk 1]
```diff
@@ -172,12 +172,20 @@ typedef void (^SwrveResourcesUpdatedListener) ();
...
+ */
+@property (nonatomic) BOOL useHttpsForEventServer;
+
 /*! Set to override the default location of the server from which Swrve will receive personalized content.
  * If your company has a special API end-point enabled, then you should specify it here.
  * You should only need to change this value if you are working with Swrve support on a specific support issue.
  */
 @property (nonatomic, retain) NSString * contentServer;
 
+/*! Use HTTPs for the in-app and user resources API endpoint.
```
Comment (marcomorain) [inline Sdk/Track/Swrve.h]: capital s in HTTPS

File: Sdk/Track/Swrve.h [hunk 2]
```diff
@@ -172,12 +172,20 @@ typedef void (^SwrveResourcesUpdatedListener) ();
  */
 @property (nonatomic, retain) NSString * eventsServer;
 
+/*! Use HTTPs for the event API endpoint.
```
Comment (marcomorain) [inline Sdk/Track/Swrve.h]: capital S in HTTPS

File: Sdk/Track/Swrve.m [hunk 3]
```diff
@@ -1420,11 +1428,11 @@ -(void) setupConfig:(SwrveConfig *)newConfig
 {
     // Set up default server locations
     if (nil == newConfig.eventsServer) {
-        newConfig.eventsServer = [NSString stringWithFormat:@"https://%ld.api.swrve.com", self.appID];
+        newConfig.eventsServer = [NSString stringWithFormat:@"%@://%ld.api.swrve.com", ((newConfig.useHttpsForEventServer)? @"https" : @"http"), self.appID];
```
Comment (marcomorain) [inline Sdk/Track/Swrve.m]: ~~ maybe make a static function bool => string ?

static NSString httpScheme(bool useHttps) {
  return useHttps ? @"https" : "http";
}

**Comment by Sergio-Mira** [comment]
@marcomorain Is this good to go?

**Pull request merged**